cfgov/unprocessed/css/molecules/archived-pill.less

```
@@ -0,0 +1,33 @@
+/* topdoc
+  name: Archived pill
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-archived-pill">
+            Archived
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-archived-pill
+  tags:
+    - cfgov-molecules
+*/
+
+.m-archived-pill {
+    padding: unit( 9px / @base-font-size-px, em ) unit( 10px / @base-font-size-px, em );
+    background-color: @teal-mid-dark;
+    border-radius: 4px;
+    color: @white;
+    position: relative;
+    display: inline-block;
+    .heading-5();
+    margin-bottom: unit( 25px / @base-font-size-px, em );
```

Should probably stick with one of our standard multiples of 15 here, unless someone really thinks those 5px make a big difference. And `heading-5()` is changing the font size, as well, so we need to compensate for that – I _think_ it can access the `@font-size` var from the `heading-5` mixin here.)

```suggestion
    margin-bottom: unit( @grid_gutter-width / @font-size, em );
```

---

This needs to be slightly off from our regular values so it's positioned correctly over an H1. The font size variable was updated in https://github.com/cfpb/consumerfinance.gov/pull/6038/commits/6aed924cf92ba897cce6715791d033a41c55bc7f.